app/__init__.py

```diff
@@ -68,4 +68,13 @@ def create_app(config_name):
     from bulk import bulk as bulk_blueprint
     app.register_blueprint(bulk_blueprint, url_prefix='/bulk')
 
+    from descriptor import descriptor as descriptor_blueprint
```

Comment:
I'm not proposing that the blueprint be called `descriptor_response` - I'm just clarifying how we should talk about the relationship between resources and descriptors. Sorry for the confusion. 

I do think it's confusing that an eponymous model exists, since a blueprint should be at a higher level of abstraction than any one model. What about the name `schema`?